### PR TITLE
shalinee | TEST | UI issue fixed

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 1]
```diff
@@ -117,6 +117,9 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
     public companyList: CompanyResponse[] = [];
     /** Stores all the menu items to be shown */
     public allItems: AllItems[] = [];
+    /** True, if sidebar needs to be shown */
+    public subdropdown;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:121]: define type of variable like boolean/string/number etc....

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 2]
```diff
@@ -363,6 +366,7 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
     public ngOnDestroy(): void {
         this.destroyed$.next(true);
         this.destroyed$.complete();
+
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts:369]: remove space

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]: Done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**